apps/internal/oauth/ops/authority/authority_test.go

```diff
@@ -320,22 +321,32 @@ func TestCreateAuthorityInfoFromAuthorityUri(t *testing.T) {
 }
 
 func TestAuthParamsWithTenant(t *testing.T) {
-	uuid1 := "00000000-0000-0000-0000-000000000000"
-	uuid2 := strings.ReplaceAll(uuid1, "0", "1")
+	uuid1 := uuid.New().String()
+	uuid2 := uuid.New().String()
 	host := "https://localhost/"
-	for _, test := range []struct {
+
+	tests := map[string]struct {
```

Comment:
>tests [](http://example.com/codeflow?start=1&length=5)

Please make sure to run this against real dSTS at least once. 

The best way to do this is to create a sample project on the following [repository](https://msazure.visualstudio.com/One/_git/EngSys-Security-dSTS?path=/Samples/Current)

Reply:
this test isn't actually firing any http requests, can you elaborate on what you mean by 'against real dSTS at least once'? I am not aware of any tests which I changed in the context of this PR are actually accessing anything e2e

Reply:
I mean if you want me to manually validate that we can get a token from dSTS using this flow I can do that (I already did that a long time ago, but the set-up we used was deprecated by the recent dSTS crack-down on MS-corp based identities)

I guess that is why you are suggesting to 'build my own test dSTS' in C# and run it locally to test against? because at this point I can't see any easy way of how to call dSTS from SAW (I can hardly compile go there), so the only thing that comes to mind is some ev2 shell extension or something

Reply:
I recommend moving this discussion outside of GitHub. Happy to cut a preview release for testing purposes. Agree that we should have some sort of manual validation, at least of some scenarios (client_credentials flow probably. I think user flows are a bit broken right now anyway)

Reply:
done